### ISSUE TITLE
Security contact

### ISSUE THREAD

**Issue by lumisota** [oaplatform/oap#206]
Is there a preferred security contact address for this project?

**Comment by alebastrov** [comment]
nope, you may create an issue if you like, someone will probably react at it
